libs/runtime/ebpf_object.h

```diff
@@ -120,6 +127,7 @@ extern "C"
     typedef void (*ebpf_zero_ref_count_t)(ebpf_core_object_t* object);
     typedef void (*ebpf_free_object_t)(ebpf_core_object_t* object);
     typedef const ebpf_program_type_t (*ebpf_object_get_program_type_t)(_In_ const ebpf_core_object_t* object);
+    typedef const bool (*ebpf_object_get_context_header_support_t)(_In_ const ebpf_core_object_t* object);
```

Comment:
Why is this needed in the base object? Is it used for more than just program objects?

Reply:
As discussed, keeping this in base object, and also exposed a `context_header_support` for map objects laos.